### PR TITLE
testing/libva-utils: upgrade to 2.4.1

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

**Comment by maxice8** [comment]
@tanertas

**Comment by tanertas** [comment]
you can take over the package for yourself.

**Comment by maxice8** [comment]
i'll pass

File: testing/libva-utils/APKBUILD [hunk 1]
```diff
@@ -1,39 +1,41 @@
...
+	meson
+	libva-dev
+	libdrm-dev
+	libx11-dev
+	libxext-dev
+	libxfixes-dev
+	wayland-dev
+	"
+source="$pkgname-$pkgver.tar.gz::https://github.com/intel/libva-utils/archive/${pkgver}.tar.gz"
 options="!check"
```
Comment (TBK) [inline testing/libva-utils/APKBUILD]:
```suggestion

```
Does the tests fail?

**Pull request closed without merging**